packages/layout/package.json

```diff
@@ -37,14 +40,12 @@
     "watch:types": "cross-env tsc --emitDeclarationOnly --declaration --declarationDir dist/types --watch --incremental"
   },
   "dependencies": {
-    "@chakra-ui/utils": "^1.5.0",
-    "@chakra-ui/vue-system": "0.1.0-alpha.1"
-  },
-  "devDependencies": {
-    "vue": ">=3.0.5"
+    "@chakra-ui/styled-system": "^1.9.0",
+    "@chakra-ui/vue-system": "0.1.0-alpha.1",
+    "@chakra-ui/vue-utils": "0.1.0-alpha.1",
+    "@chakra-ui/vue-layout": "0.1.0-alpha.1"
```

Comment:
This might be a circular dependency. The `@chakra-ui/vue-layout` package is requiring itself. 🤔

Reply:
it was late man :D fixed